frontend/src/metabase/auth/components/Login/Login.tsx

```diff
@@ -25,26 +20,27 @@ interface LoginProps {
 }
 
 export const Login = ({ params, location }: LoginProps): JSX.Element => {
+  const { classes } = useStyles();
   const providers = useSelector(getAuthProviders);
   const selection = getSelectedProvider(providers, params?.provider);
   const redirectUrl = location?.query?.redirect;
 
   return (
     <AuthLayout>
-      <LoginTitle>{t`Sign in to Metabase`}</LoginTitle>
+      <div className={classes.title}>{t`Sign in to Metabase`}</div>
       {selection && selection.Panel && (
-        <LoginPanel>
+        <div className={classes.panel}>
           <selection.Panel redirectUrl={redirectUrl} />
-        </LoginPanel>
+        </div>
       )}
       {!selection && (
-        <ActionList>
+        <div className={classes.actionList}>
```

Comment:
Why not use `ul` and `li` as it is a list of actions?

Reply:
@losrebellos Mind expanding on why you'd prefer `Stack` in this context?

Reply:
I think generally Stack is better here but I decided not to do any markup changes in the PR if possible

Reply:
@kdoh Stack is literally made for listing/stacking items vertically with a consistent spacing (see Group for horizontal).
Box seems to be used when in need of a "div" with some specific styling.